lib/src/DatePicker/DatePicker.tsx

```diff
@@ -34,12 +46,25 @@ export interface BaseDatePickerProps {
   initialFocusedDate?: DateType;
 }
 
-export interface DatePickerProps extends BaseDatePickerProps {
+export interface DatePickerProps
+  extends BaseDatePickerProps,
+    WithStyles<typeof styles>,
+    WithUtilsProps {
   date: MaterialUiPickersDate;
   onChange: (date: MaterialUiPickersDate, isFinished?: boolean) => void;
 }
 
-export class DatePicker extends React.PureComponent<DatePickerProps & WithUtilsProps> {
+interface DefaultBaseDatePickerProps {
```

Comment:
I do not understand prospective of this interface

Reply:
Oh, now when tests failed I see why, that hack was added